app/data_providers/dispatch.py

```diff
@@ -25,6 +27,27 @@
 from app.models.request_user import RequestUser
 from app.rankers.algorithms import rank_by_preferred_topics, spread_topics
 
+# todo: add thompson sampling
+class Item2ItemDispatch:
+
+    def __init__(self,
+                 # metrics_client: MetricsFetchable,
+                 item_recommender: Item2ItemRecommender):
+        # self.metrics_client = metrics_client
+        self.item_recommender = item_recommender
+
+    async def syndicated(self, resolved_id: int, count: int) -> List[CorpusRecommendationModel]:
+        recs = await self.item_recommender.syndicated(resolved_id, count)
+        # ranked = self.metrics_client.rank_items(recs, rankers=[thompson_sampling_28day])
+        # return ranked
```

Comment:
Suggestion: Use a git branch to store future work. Considering it's just a few lines I don't feel super strongly, so up to you.